Formula/t/titan-server.rb

```diff
@@ -26,6 +21,12 @@ class TitanServer < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "c6d98726b834d82fe0adb786919652c5f0e0974ff3cb03969b2c69042cd4998c"
   end
 
+  # upstream is not responsive on the issues and no commits since 2015 dec
+  # community has forked the project to janusgraph
+  # https://github.com/thinkaurelius/titan/issues/1393
+  # https://github.com/thinkaurelius/titan/issues/1392
+  disable! date: "2024-09-12", because: :unmaintained
```

Comment:
```suggestion
  disable! date: "2025-01-01", because: :unmaintained
```
Let's have a short deprecation period here.